### ISSUE TITLE
Add model checks on the parameters

### ISSUE THREAD

**Issue by GaetandeCast** [probabl-ai/skore#2670]
Implement the following checks:

|Failure | Description | Typical signal |
|---|---|---|
|Important Hyperparams untuned|Some hyper parameters (e.g. learning rate) should be tuned for optimal performance|Important hyper parameter left at default value in the param_grid of a GridSearchCV estimator.|
|Hyperparams selected are at the edge|The gridsearch has selected a param at the edge of the range|Selected param value is min or max of range|
|Lack of threshold choice|Default decision threshold used despite asymmetric costs|Default threshold (e.g. 0.5), no evidence of tuning vs cost|

**Comment by direkkakkar319-ops** [comment]
Hi @GaetandeCast  I would love to take a stab at implementing the three model checks described here. culd you assign this issue to me

I'll start by looking at how existing checks are structured in the codebase (via the parent issue #2617) to make sure my implementation fits the pattern.

**Comment by glemaitre** [comment]
@direkkakkar319-ops This issue is not the best to be looked at for the moment. We are still not sure about the API. So it would be better to take a stab at something else.

**Comment by direkkakkar319-ops** [comment]
Understood @glemaitre, thanks for the heads up. I will look for another issue to work on.

**Comment by GaetandeCast** [comment]
Closing this as the "Lack of threshold choice" check is not so obvious to implement. We can revisit it later.
